packages/cozy-harvest-lib/src/components/KonnectorAccounts.jsx

```diff
@@ -8,17 +8,18 @@ import { translate } from 'cozy-ui/transpiled/react/I18n'
 import Infos from 'cozy-ui/transpiled/react/Infos'
 import Button from 'cozy-ui/transpiled/react/Button'
 import get from 'lodash/get'
+import compose from 'lodash/flowRight'
```

Comment:
Nit but I find it strange to rename `flowRight` to `compose`

Reply:
Compose is the original name of this function. It was renamed into flowRight for parity with the flowLeft function in later versions of lodash (in lodashFP, it is named compose BTW). Personally I find it better named as compose as it makes reference to [function composition](https://en.wikipedia.org/wiki/Function_composition).

Reply:
> I use this name since the beginning I use compose at Cozy. I thought it was understood as a convention.

I know and it's understood, no worries. I just hadn't come across it in a while so I thought I mention it in passing.